src/libs/actions/Report.ts

```diff
@@ -1462,6 +1468,133 @@ function toggleSubscribeToChildReport(childReportID = '0', parentReportAction: P
     }
 }
 
+function updatePolicyReportName(reportID: string, value: string) {
```

Comment:
Let's rename this to updateReportName (since we don't actually reference the Policy anywhere within this method)

Reply:
Done!